src/Apps/Order/Routes/Payment/context.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useReducer, createContext, Dispatch } from "react"
+import { paymentReducer } from "./reducer"
+import { PaymentAction, PaymentState } from "./types"
+
+const initialPaymentState = {
+  selectedBankAccountId: "",
+}
+
```

Comment:
for now I just added a single property to the state and it is **not** used in the actual app. 

if we want to adopt this approach, this would be the place where `useState` definitions would be translated into.